### PR TITLE
Implement clineshift score

### PR DESCRIPTION
**Pull request by Adriel-M**

### CODE REVIEW

**Review by luvicky — APPROVED** [review]
Looks good 👍

File: R/clineshiftScore.R [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+#' @param testMapB A numeric vector for the distance map of test sequence B.
+#' @param epsilon A scoring parameter. Defaults to 0.2.
+#' @return The clineshift score between two sequences.
+#'
+#' @examples
+#' refA <- c("A", "V", "T", "-", "-", "D")
+#' refB <- c("A", "V", "T", "F", "F", "D")
+#' testA <- c("A", "V", "-", "-", "T", "D")
+#' testB <- c("A", "V", "T", "F", "F", "D")
+#' refMap <- generatePairMap(refA, refB)
```
Comment (luvicky) [inline R/clineshiftScore.R:23]: generatePairMap(refA, testA?)

**Review by luvicky — COMMENTED** [review]
Line 23

File: R/clineshiftScore.R (hunk 1 above)
Comment (Adriel-M) [reply R/clineshiftScore.R:23]: Pair maps are between two aligned sequences.

**Comment by Adriel-M** [comment]
Still need to add the alignment examples.

EDIT: included

File: R/clineshiftScore.R (hunk 1 above)
Comment (Adriel-M) [reply R/clineshiftScore.R:23]: On the other hand, line 24 was wrong.

**Pull request merged**